spec/unit/message_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'message'
+
+describe Message do
+  describe '.all' do
+    it 'returns message' do
+      Message.create(message: 'This is the a message')
+
+      message = Message.all
+
+      expect(message.first).to be_a Message
```

Comment:
Do you really care if the object is of type Message?  What the program were expanded to include some other type of post?  Maybe you only really care that they all behave in the right way...